contracts/validator-manager/ValidatorManager.sol

```diff
@@ -25,6 +21,8 @@ import {ContextUpgradeable} from
     "@openzeppelin/contracts-upgradeable@5.0.2/utils/ContextUpgradeable.sol";
 import {Initializable} from
     "@openzeppelin/contracts-upgradeable@5.0.2/proxy/utils/Initializable.sol";
+import {IACP99SecurityModule} from "./interfaces/IACP99SecurityModule.sol";
```

Comment:
Is this used anywhere?  And I also notice that this isn't `IACP99ValidatorManager`. Might make sense to just add a comment saying why (I'm guessing because it's not worth the adding all the events that aren't specified in the interface).